modules/tinymce/src/core/main/ts/api/NotificationManager.ts

```diff
@@ -153,6 +154,11 @@ const NotificationManager = (editor: Editor): NotificationManager => {
         getImplementation().close(notification);
       });
     });
+
+    editor.addShortcut('alt+F12', 'focus on the first notification', () =>
```

Comment:
```suggestion
    editor.addShortcut('alt+F12', 'Focus to the first notification', () =>
```
Might as well keep it the same as the help dialog... not that we use these descriptions anywhere

https://github.com/tinymce/tinymce/blob/dcbb28a3909bafee4e26f06d4336f99db568ff13/modules/tinymce/src/plugins/help/main/ts/data/KeyboardShortcuts.ts#L23